module/README.md

```diff
@@ -31,6 +31,39 @@ async def root():
     return {"message": "Hello World"}
 ```
 
+# Prometheus Metrics Middleware
+
+Here's how you can use the middleware to define your own custom metrics, which the otel will scrape on the `/metrics` endpoint added to your app automatically.
```

Comment:
I think it might be good to add more context here. I don't expect people to know what otel is, what the metrics are for, or why they'd need an endpoint.

Questions to answer in here:

1. What are these metrics for, and where will I be able to see them?
2. What's otel? You can either name it and explain it, link to an existing doc for it, or remove it and just say something about how this exposes an endpoint that we use to pull metrics, not mentioning the tooling at all.
3. How do you actually define and record a metric? Linking to the prometheus Concepts docs might be useful. Things like the metric types doc here: https://prometheus.io/docs/concepts/metric_types/ or the one about the data model.
4. The pull model is a bit nonintuitive, so it'd be good to be clear that the middleware stores metrics data, and serves it to the prc metrics service on request. So we'll poll this endpoint periodically to get the metrics.